cpp/src/arrow/filesystem/azurefs_test.cc

```diff
@@ -113,33 +123,6 @@ AzuriteEnv* GetAzuriteEnv() {
 
 // Placeholder tests
 // TODO: GH-18014 Remove once a proper test is added
-TEST(AzureFileSystem, UploadThenDownload) {
```

Comment:
Shouldn't the comment above be removed as well?

Reply:
I kept it because I consider `InitializeCredentials` to also be a placeholder test. That was created just to make sure the Azure SDK build was working and I expect it to be deleted once we add support for initialising the `AzureFileSystem` with all the different types of auth.